src/main/groovy/org/jenkinsci/plugins/globalEventsPlugin/GlobalItemListener.java

```diff
@@ -0,0 +1,86 @@
+package org.jenkinsci.plugins.globalEventsPlugin;
+
+import hudson.Extension;
+import hudson.model.Item;
+import hudson.model.listeners.ItemListener;
+import jenkins.model.Jenkins;
+
+import java.util.HashMap;
+import java.util.logging.Logger;
+
+/**
+ * Created by Renjith Pillai (i306570) on 15/09/16.
+ */
+@Extension
+public class GlobalItemListener extends ItemListener
+{
+    protected static Logger log = Logger.getLogger(GlobalItemListener.class.getName());
+
+    public GlobalItemListener() {
+        log.fine(">>> Initialised");
+    }
+
+    GlobalEventsPlugin.DescriptorImpl parentPluginDescriptorOverride = null;
+
+    GlobalEventsPlugin.DescriptorImpl getParentPluginDescriptor() {
+        if (parentPluginDescriptorOverride != null){
+            return parentPluginDescriptorOverride;
+        } else {
+            return Jenkins.getInstance().getPlugin(GlobalEventsPlugin.class).getDescriptor();
+        }
+    }
+
+    @Override
+    public void onCreated(final Item item)
+    {
+        this.getParentPluginDescriptor().processEvent(Event.ITEM_CREATED, log, new HashMap<Object, Object>() {{
+            put("item", item);
+        }});
+        super.onCreated(item);
```

Comment:
Are you sure that this is needed? I'm expecting that same event would be handled by different `ItemListener` instances anyway.